.pre-commit-config.yaml

```diff
@@ -3,16 +3,18 @@ repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v2.4.0
     hooks:
-      - id: flake8
-        # line too long and line before binary operator (black is ok with these)
-        types:
-          - python
       - id: trailing-whitespace
       - id: end-of-file-fixer
       - id: check-yaml
         args: ["--unsafe"]
       - id: check-added-large-files
   - repo: https://github.com/psf/black
-    rev: 19.10b0
+    rev: 23.7.0
```

Comment:
Was there a  reason for bumping it up? I'm on board with it, but just wanted to check if you hit an error / any issues with the old version (since it'll help to know when migrating other packages!)

Reply:
Yeah I hit an error without it.  I can't remember exactly what it is but happy to revert and report the error if you like

Reply:
No need to change! I'll keep an eye out for errors stemming from this in other packages 😬